src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTupleTest.cs

```diff
@@ -306,6 +335,101 @@ .maxstack  4
 }
 ");
         }
+        [Fact]
+        public void TupleTypeDeclaration()
+        {
+            var source = @"
+class C
+{
+    static void Main()
+    {
+        (int, string, int) x = (1, ""hello"", 2);
+        System.Console.WriteLine(x.ToString());
+    }
+}
+
+" + trivial3uple;
+
+            var comp = CompileAndVerify(source, expectedOutput: @"{1, hello, 2}");
+        }
+
+        [Fact]
+        public void TupleTypeMismatch()
+        {
+            var source = @"
+class C
+{
+    static void Main()
+    {
+        (int, string) x = (1, ""hello"", 2);
+    }
+}
+" + trivial2uple + trivial3uple;
+
+            // PROTOTYPE there should be no mention of Item1, Item2 or Item3 in there error for this test
```

Comment:
Perhaps you need to clarify rational behind this suggestion. Would we then lose an easy way to verify field names assigned during binding? How would we find tests that would have to be adjusted to keep them serving their purpose?